src/NServiceBus.Core/ObjectBuilder/ServiceCollectionExtensions.cs

```diff
@@ -9,7 +9,7 @@
     /// Contains extension methods for <see cref="IServiceCollection"/> that were formerly provided by <see cref="IConfigureComponents"/>.
     /// </summary>
     [ObsoleteEx(
-        Message = "Use methods on IServiceCollection instead.",
+        Message = "Use methods on IServiceCollection instead and ensure to explicitly register all interfaces of a specific type as this will no longer be done automatically.",
```

Comment:
```suggestion
        Message = "Use methods on IServiceCollection instead. Note that the `Add...` extension methods do not register interfaces implicitly.",
```

maybe make it a separate sentence?